draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -275,6 +294,9 @@ happen through Unicast DNS as noted in {{RFC6762}}, Section 3.
 An ICE agent SHOULD ignore candidates where the hostname resolution returns
 more than one IP address.
 
+When resolving a mDNS name, the ICE agent SHOULD set the unicast-response bit of
```

Comment:
I would move this to Step 2 of the algorithm above. Suggest also giving some supporting rationale.

Reply:
ok